packages/core/src/controllers/relayer.ts

```diff
@@ -253,7 +253,7 @@ export class Relayer extends IRelayer {
   }
 
   private rejectTransportOpen() {
-    throw new Error("closeTransport called before connection was established");
+    throw new Error("Attempt to connect to relay via `transportOpen` has stalled.");
```

Comment:
```suggestion
    throw new Error("Attempt to connect to relay via `transportOpen` has stalled. Retrying...");
```

As Derek suggested, should we mention that auto retry is active?